fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,55 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --env=ALSA_CONFIG_PATH=
```

Comment:
so you removed the `_PATH` for the various plugins. But they aren't set anywhere else.

Reply:
What would be the best way to do this?